### DISCUSSION TITLE
AOSP 12.0 v400.g

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2133] [General]
- January security patch level

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v400.g'>AOSP 12.0 v400.g</a>.</em>

**Comment by phhgsi** [comment]
Good bro you are great

**Comment by HeroRickyGAMES** [comment]
In the Christmas version I tested on the Moto G6 ALI was facing a bug like the SIM not working, this was the only bug I found testing this GSI being the only one that booted the Moto G6 ALI. I thought it could be the Vendor, I even modified the base Vendor, but without success.

**Comment by NeonzGamingYT** [comment]
Need fix for gogapps variant recent key :D

**Comment by lovefaruk** [comment]
I can't boot 400.f or 400.g in my Galaxy A70 with Android Q or Android R? 
Please Helpt

**Comment by phhusson** [comment]
Are you using magisk? Magisk is known to break gsis

**Reply by lovefaruk** [reply]
400.e is working very nice... Thank you for that version.
with magisk or without magisk.. no luck booting(400.f or 400.g)
Here's how I did:
Installed Full Stock ROM Android 11 DUI2 (Galaxy A70GM)
Installed TWRP 3.6 official version
Format Data - yes (Inside TWRP)
Install -Disable_Dm-Verity_ForceEncrypt
Install - 400.g gsi image
restart 
Got Boot Loop 
No luck in booting it up.. 
Please help

**Reply by lovefaruk** [reply]
just got booted. 400.g with Christmas tree boot-screen 
Thanks.. Flashed with Latest FULL STOCK ROM.. A705GMxxxxDUI2

**Reply by lovefaruk** [reply]
anyway to root this version..????

**Reply by devrieshh** [reply]
aint all the versions rooted exept the securised ones?

**Comment by Taogezuibang** [comment]
Hello, I am from China. When using the ROM you developed, I found that the screen fingerprint could not be used properly. To enter on the highest brightness, it is not sensitive. When Volte is turned on, calls can be made but calls cannot be answered because the incoming phone does not respond. There are also warm color modes that will blink. We look forward to your solution. Thank you.

**Reply by KawaiiHachimi** [reply]
Add a description of your device pls.

**Reply by KawaiiHachimi** [reply]
I have the same problem with my Redmi K30 Ultra (Mediatek MT6885 Dimensity 1000+). Default FOD color on MIUI is white.

**Comment by RendeRR** [comment]
Hi, this is a log of DNS requests from a device with this ROM.  What is this address and how is it used?
![Screenshot_20220111-143357](https://user-images.githubusercontent.com/790707/148935854-97d750c6-aacd-4950-aad2-10e4ebed7a7c.png)

**Reply by RendeRR** [reply]
Every ~10 minutes. 😳

**Reply by phhusson** [reply]
Uh thanks for the report, this shouldn't happen on v400.g. Are you really running my v400.g?

Did you tick "allow takeover by phh" in treble settings?

For full disclosure, this hostname is used for the "Allow remote takeover by phh" which allows me to remotely adb connect into users' device. As you can check by yourself with https://github.com/phhusson/device_phh_treble/blob/android-12.0/remote/phh-remotectl.rc there shouldn't be any such name resolution unless the option is ticked. To the best of my knowledge, this feature is totally broken on  Android 12 because `dbclient` is not  installed. So I can't see how it is possible for my v400.g to ping this hostname

**Reply by RendeRR** [reply]
The setting has been enabled. Turned it off and requests stopped. System information. Is it 400g?
![Screenshot_20220111-200123](https://user-images.githubusercontent.com/790707/148990763-1482cc4c-69c9-4bc9-83b5-e769a14b4245.png)

**Reply by AvrahamTsaban** [reply]
No, it's a build from 05.12.2021

**Reply by dayona** [reply]
> Uh thanks for the report, this shouldn't happen on v400.g. Are you really running my v400.g?
> 
> Did you tick "allow takeover by phh" in treble settings?
> 
> For full disclosure, this hostname is used for the "Allow remote takeover by phh" which allows me to remotely adb connect into users' device. As you can check by yourself with https://github.com/phhusson/device_phh_treble/blob/android-12.0/remote/phh-remotectl.rc there shouldn't be any such name resolution unless the option is ticked. To the best of my knowledge, this feature is totally broken on Android 12 because `dbclient` is not installed. So I can't see how it is possible for my v400.g to ping this hostname

Is it possible to view my device screen or access adb on my local network or from a remote server like android-remote.phh.me?
How can I set this up at my end so that I can access my device from anywhere, is the code for hosting android-remote.phh.me is public?

**Comment by ghost** [comment]
How can I fix no sound 
I tried increasing volume still no luck .
Power button and vol button
Doesn't work sometimes .
Random reboot.

Any fix.

**Comment by Skulking12** [comment]
Hi Why u don't add Dynamic theme in this gsi 
The 12L gsi Its have theme with wallpaper but in ur gsi don't have 
And Add Wallpaper and style in gsi

**Reply by phhusson** [reply]
Because 12L has no source code yet.

**Comment by aryan0078** [comment]
No volte on Exynos 9611 device - m31 
Bt call audio not working media works btw

**Comment by phhusson** [comment]
yes

Le ven. 14 janv. 2022 à 17:40, devrieshh ***@***.***> a
écrit :
